changelog.d/pse-329.feature

```diff
@@ -0,0 +1 @@
+Using the same User Avatar for all messages in the timeline
```

Comment:
We should change the name of this file to point to the FOSS issue I created and linked.

Reply:
Done.